scripts/src/models/orders.ts

```diff
@@ -227,14 +227,30 @@ export const Order = {
 		return query;
 	},
 
-	getAll<T extends Order>(filters: GetOrderFilters & { userId: string }, limit?: number): Promise<T[]> {
-		const query = this.genericGet(filters);
+	/**
+	 * Gets orders by `filters` (by `user_id`) object, maps it to order ids and searching  by `IN order_ids`
+	 * First `genericGet` will be replaced by caching lookup
+	 *
+	 * @param      {GetOrderFilters & {userId: string}}  filters
+	 * @param      {number}  limit
+	 * @return     {Promise<T[]>}  filtered orders including p2p
+	 */
+	async getAll<T extends Order>(filters: GetOrderFilters & { userId: string }, limit?: number): Promise<T[]> {
+		const allOrders = await this.genericGet(filters).getMany(); // can be replaced by cache
+
+		const ids: string[] = allOrders.map((o: Order) => o.id);
```

Comment:
do you need to specify the type?
doesn't the compiler infer that on it's own?
that is:
`const ids = allOrders.map((o: Order) => o.id);`

Reply:
Done